app/models/actor.rb

```diff
@@ -51,11 +51,9 @@ class Actor < ApplicationRecord
             uniqueness: {
               case_sensitive: false,
               allow_nil: true
-            }
-
-  validates :psu_id,
+            },
             presence: true,
-            on: :from_omniauth
+            unless: -> { orcid.present? }
```

Comment:
We have the same situation here as we did on `orcid`. I think we need to separate to the uniqueness validation to run all the time, and the presence validation to run `unless orcid.present?`

Reply:
yep, sure enough!